src/modules/utilities/utilities.test.ts

```diff
@@ -0,0 +1,74 @@
+// Override system date
+jest.useFakeTimers().setSystemTime(new Date(1675298702433));
+// Override Math.random() to always return the same value
+jest.spyOn(global.Math, 'random').mockReturnValue(0.123456789);
+
+import SeededRand from './SeededRand';
+import { AchievementType } from '../GameConstants';
+import Rand from './Rand';
+
+describe('Test SeededRand & Rand', () => {
+    it('Test rand', () => {
+        expect(Rand.next()).toBe(0.123456789);
+    });
```

Comment:
Not sure that this is a useful test, it is just returning the value you are mocking on Math.random

Reply:
It's only for coverage, not sure if there's a better way to test this without just keeping Math.random as a random value and saying it should be between `1` and `0`?

Reply:
I'm fine leaving it as is just for the coverage

Reply:
Actually, if I just test a `Rand.intBetween` or something like that, it should automatically cover this function too.

Reply:
Yeah that sounds like a good idea :+1: